### PR TITLE
Added First html file

### PR DESCRIPTION
**Pull request by sandpkasam**
First Html file added

### CODE REVIEW

**Review by udaymattam — COMMENTED** [review]
Add a more context...

File: Basic/First.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Class</title>
+  </head>
+  <body>
+    <p>
```
Comment (udaymattam) [inline Basic/First.html:7]: Add at least 2 or 3 <p> tags

Comment (sandpkasam) [reply Basic/First.html:7]: ok i will do that

Comment (sandpkasam) [reply Basic/First.html:7]: Resolved commit 61e4c6f028d765177559bb4c9b1c1f37c8084402

**Review by udaymattam — APPROVED** [review]

**Pull request merged**